### PR TITLE
Move the plugin docs to the fastlane docs page

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Review by taquitos — APPROVED** [review]
🎈🍬🐐

File: fastlane/docs/Plugins.md [hunk 1]
```diff
@@ -1,156 +1,3 @@
...
-- Edit the `lib/fastlane/plugin/[plugin_name]/actions/[plugin_name].rb` and implement your action
-- Easily test the plugin locally by running `fastlane add_plugin` in your project's directory and specifying the local path when asked for it
-
-##### New plugin for existing gem
-
-If you already have an existing gem you want to provide a `fastlane` plugin for, you'll still have to create a new Ruby gem. The reason for that is the way plugins are imported.
-
-The example project [xcake](https://github.com/jcampbell05/xcake) contains a folder named `fastlane-plugin-xcake`.
-
-All you have to do if you have an exiting gem:
```
Comment (taquitos) [inline fastlane/docs/Plugins.md]: I like how this typo was fixed already in the docs.

**Comment by KrauseFx** [comment]
Yeah, we have the best contributors :)

**Pull request merged**